docs/developer-guide/pull-requests.md

```diff
@@ -6,18 +6,19 @@ Rules of thumb:
 -   Review against the [Developer Guide criteria](rules.md).
 -   Assign one or more of the appropriate [`PR: needs *` labels](https://github.com/stylelint/stylelint/labels) when requesting a change.
 -   Resolve conficts by [rebasing](https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase), rather than merging the target branch into the pull request branch.
--   Merge simple documentation fixes after one approval and all other pull requests after two approvals.
 
 Merge process:
 
-
-1.  "Squash and merge" commits, ensuring the resulting commit message is consistently formatted:
+1.  Make sure the pull request has the correct number of approvals:
+    -   For simple documentation fixes: one
+    -   For all other pull requests: two
```

Comment:
I think punctuation marks are missing. I'm not sure, but maybe the following ones are correct:

> -   For simple documentation fixes: one;
> -   For all other pull requests: two.